docs/deviceUsecase/deviceBgpUsecase.md

```diff
@@ -0,0 +1,260 @@
+# Scenario-1: Simple BGP and Route
+```python
+device = config.devices.device(name="d1")[-1]
+eth = device.ethernets.ethernet(name='eth1', port_name="p1")[-1]
+eth.ipv4_addresses.ipv4(name="ip1")
+
+bgp = device.bgp
+bgp.router_id = "1.1.1.1"
+bgp_int = bgp.ipv4_interfaces.add(ipv4_name="ip1")
+bgp_peer = bgp_int.peers.add(name="bgp1")
+v4_routes = bgp_peer.v4_routes.add(name="route1")
```

Comment:
is there an example of how to specify the bgp_peer's IP address?

Reply:
I have added required property related to bgp_peer